docs/search_relevance/query_design.md

```diff
@@ -28,11 +28,68 @@ Each query has:
 
 ## Intentions & Expectations
 
+### Phrase matching
+
+| Status | Query name | Ranking evaluation test |
+| :--- | :--- | :--- |
+| TODO | PhraseMatchQuery | TBD |
+
+#### Intentions
+
+Searching for an exact, ordered set of tokens, in quotation marks
+
+#### Data features
+
+* `data.title`
+* `data.alternativeTitle`
+* `physicalDescription`
+* `subjects.label`
+* `genres.label`
+* `contributors.label`
+* `description`
+
+#### Expectations
+
+* Searching for a phrase in quotation marks should match works containing that exact set of tokens, in the same order
```

Comment:
When using the word phrase do we not already infer order?

Maybe
```suggestion
* Phrases are defined by tokens wrapped in quotation marks and should be matched exactly
```
Or is that too lose?